### ISSUE TITLE
Pyright doesn't show all errors at once. Only a small portion. Pyright error limit

### ISSUE THREAD

**Issue by Day0Dreamer** [RobertCraigie/pyright-python#174]
Pyright keeps on is giving more and more errors after previous ones have been dealt with. This is unusual, as with other tools like mypy flake8 or similar, you are presented with the whole listing of errors.

Basically, it feels as if there is some kind of limit. Most likely "just show 2-3 .py files and omit the rest".

Here is the flow:
- Start pyright --watch
- Observe 20 errors in two files
- Spend time fixing errors
- Observe errors go down to 0
- Nothing happens until I Ctrl+C and restart pyright --watch
- Observe 20 errors in two yet untouched files
- Continue working

Sidenote: --watch flag doesn't change the number of errors, or the files that are displayed.

Config:
```
[tool.pyright]
include = ["src"]
ignore = ["tests/**/*", "src/ui.py", "src/sendmetoc4d.py"]
pythonVersion = '3.10'
pythonPlatform = 'All'
useLibraryCodeForTypes = true
venvPath = "C:\\Users\\DayDreamer\\AppData\\Local\\hatch\\env\\virtual\\packer\\Xm-Cv8OZ"
venv = "packer"
```

**Comment by RobertCraigie** [comment]
Sorry for the delayed response here, I don't think I've personally seen this behaviour before for what it's worth but the type checking output isn't implemented in this repository, if you're still seeing this behaviour please ask for help in the pyright repo https://github.com/microsoft/pyright
